interpreter/cling/tools/plugins/clad/CMakeLists.txt

```diff
@@ -27,6 +27,7 @@ if(MSVC)
     CMAKE_ARGS -G ${CMAKE_GENERATOR} -DCLAD_BUILD_STATIC_ONLY=ON
                -DCMAKE_INSTALL_PREFIX=${CLING_PLUGIN_INSTALL_PREFIX}
                -DCLAD_PATH_TO_LLVM_BUILD=${LLVM_BINARY_DIR}
+               -DCLANG_INCLUDE_DIRS=${CLANG_INCLUDE_DIRS}
```

Comment:
What happens after installation when clang is builtin? Won't there be a hard-coded include to the build directory in this case?